core/src/main/scripts/importer/validateData.py

```diff
@@ -3612,6 +3618,19 @@ def validate_defined_caselists(cancer_study_id, case_list_ids, file_types, logge
                 "case list will be selected by default when both mutation and CNA data are available.",
                 cancer_study_id + '_cnaseq')
 
+def validateMetadata(metadata_file_path, logger):
+        """Validate the metadata file."""
+        logger.debug('Starting validation of YAML file',
+        	extra={'filename_': metadata_file_path})
+        with open(metadata_file_path, 'r') as stream:
+            try:
+                parsedYaml = yaml.load(stream)
+                logger.info('Validation of YAML file complete.',
+                extra={'filename_': metadata_file_path})
+            except yaml.YAMLError as exc:
+                logger.error('The file provided is not in YAML format',
```

Comment:
if documentation states YAML/JSON, then it is perhaps also better to state it here, just for consistency.